test/client/spec/components/victory-line.spec.jsx

```diff
@@ -1,21 +1,126 @@
 /**
  * Client tests
  */
-
+import React from "react";
+import ReactDOM from "react-dom";
+import VictoryLine from "src/components/victory-line";
 // Use `TestUtils` to inject into DOM, simulate events, etc.
 // See: https://facebook.github.io/react/docs/test-utils.html
+import TestUtils from "react-addons-test-utils";
+
+const getElement = function (output, tagName) {
+  return ReactDOM.findDOMNode(
+    TestUtils.findRenderedDOMComponentWithTag(output, tagName)
+  );
+};
+
+let renderedComponent;
+
+describe("components/victory-line", () => {
+  describe("default component rendering", () => {
+    before(() => {
+      renderedComponent = TestUtils.renderIntoDocument(<VictoryLine/>);
+    });
+
+    it("renders an svg with the correct width and height", () => {
+
+      const svg = getElement(renderedComponent, "svg");
+      // default width and height
+      expect(svg.style.width).to.equal(`${VictoryLine.defaultProps.width}px`);
+      expect(svg.style.height).to.equal(`${VictoryLine.defaultProps.height}px`);
+    });
+  });
+
+  describe("rendering with null data", () => {
+    it("renders one line segment when there is no null data", () => {
+      const data = [
+        {x: 1, y: 1},
+        {x: 2, y: 4},
+        {x: 3, y: 5},
+        {x: 4, y: 2},
+        {x: 5, y: 3},
+        {x: 6, y: 4},
+        {x: 7, y: 6}
+      ];
+      renderedComponent = TestUtils.renderIntoDocument(<VictoryLine data={data}/>);
+      const path = TestUtils.scryRenderedDOMComponentsWithTag(renderedComponent, "path");
+      expect(path.length).to.equal(1);
+    });
+
+    it("renders two line segments when there are two continuous sections of data", () => {
```

Comment:
data sorted by x... I guess it's discrete values, but we would be drawing a continuous line except for the null discontinuity.